save-plugins/fix-plugin/src/commonMain/kotlin/com/saveourtool/save/plugins/fix/FixPlugin.kt

```diff
@@ -66,22 +67,20 @@ class FixPlugin(
     private lateinit var extraFlagsExtractor: ExtraFlagsExtractor
 
     @Suppress("TOO_LONG_FUNCTION")
-    override fun handleFiles(files: Sequence<TestFiles>): Sequence<TestResult> {
+    override fun handleFiles(evaluatedToolConfig: EvaluatedToolConfig, files: Sequence<TestFiles>): Sequence<TestResult> {
         testConfig.validateAndSetDefaults()
         val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
         val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().single()
         extraFlagsExtractor = ExtraFlagsExtractor(generalConfig, fs)
 
         return files.map { it as FixTestFiles }
-            .chunked(fixPluginConfig.batchSize!!.toInt())
+            .chunked(evaluatedToolConfig.batchSize)
             .map { chunk ->
+                val copyPaths = chunk.map { it.test }
```

Comment:
this part looks very similar to `com.saveourtool.save.core.utils.CmdExecutorBase`, probably it can be reused here